docker-compose.yml

```diff
@@ -39,8 +39,10 @@ services:
       PS_DEV_MODE: ${PS_DEV_MODE:-0}
       ADMIN_MAIL: ${ADMIN_MAIL:-demo@prestashop.com}
       ADMIN_PASSWD: ${ADMIN_PASSWD:-Correct Horse Battery Staple}
+      XDEBUG_MODE: ${PS_XDEBUG_MODE:-off}
+      XDEBUG_CONFIG: ${PS_XDEBUG_CONFIG:-client_host=host.docker.internal}
```

Comment:
So we can update the docker image to auto install xdebug and set it up according to the users preferences (and OS).

Reply:
Ok, where does `host.docker.internal` come from ? 
Also, please note that you can still create a `docker-compose.override.yaml` for your local overrides

Reply:
the host.docker.internal is a standard domain used by docker images to reflect the host machine. If you set XDebugs client_host to that value, XDebug can connect to the host machine. Otherwise XDebug will not work.